components/form-file-button/index.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Should we call this `FileUpload`? I don't think I would have found it easily with this name.